### ISSUE TITLE
Prepare XED

### ISSUE THREAD

**Issue by RyanHam04** [RyanHam04/crosslingual-emotion-llm#3]
- Download XED dataset: start with English as a baseline
- Later we extend to a total of 4 languages 
- Inspect data
- Decide train/dev/test sets
- Build a dataloader
- Label the data (See [READ.ME XED-dataset](https://github.com/Helsinki-NLP/XED/tree/master?tab=readme-ov-file#format)) and parse as multi-labeled vectors
